rubocop-rspec.yml

```diff
@@ -213,20 +213,8 @@ RSpec/VoidExpect:
 RSpec/Yield:
   Enabled: true
 
-Capybara/CurrentPathExpectation:
-  Enabled: true
-
 RSpec/Capybara/FeatureMethods:
```

Comment:
This looks like it was missed out, but this cop remains in `rubocop-rspec`, and will be turned into a more generic, non-Capybara specific cop.